platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -44,7 +46,7 @@
    *
    * <p>1.0 - initial version
    */
-  private static final String CURRENT_VERSION = "1.0";
+  private static final String CURRENT_VERSION = "1.1";
```

Comment:
❗️ We should document what has changed in 1.1 above.

Reply:
Reverted this change so that the migratable versions are remaining the same.